src/features/rewards/box/index.tsx

```diff
@@ -32,6 +33,7 @@ export interface Props {
   description?: string
   toggle?: boolean
   checked?: boolean
+  onReview?: () => void
```

Comment:
As far as I can see, the Box component is pretty generic. So 'onReview' is a bit confusing because not every box will be reviewable, and the box itself isn't for review. Instead, could the box component accept props about the alert it should display, if any. For example:
`alertData: { onReview, alertTitle, alertDescription, alertAction }`

Actually in this case, since it's so specific and we have a custom component just for this alert, why not have the alert component passed through as a prop?

`attachedAlert: React.ReactNode`

Reply:
sure I like `attachedAlert: React.ReactNode`

Reply:
done 👍